src/test/java/com/wikia/webdriver/testcases/discussions/CreatingPostTests.java

```diff
@@ -41,12 +43,65 @@ public void anonUserOnMobileCanNotWriteNewPost() {
   @Test(groups = "discussions-anonUserOnDesktopCanNotWriteNewPost")
   @Execute(asUser = User.ANONYMOUS)
   @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
-
   public void anonUserOnDesktopCanNotWriteNewPost() {
     userOnDesktopMustBeLoggedInToUsePostCreator();
   }
 
   /**
+   * LOGGED-IN USERS ON DESKTOP SECTION
+   */
+
+  @Test(groups = "discussions-loggedInUsersDesktopPosting")
+  @Execute(asUser = User.USER)
+  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+  public void loggedInUserCanExpandPostEditorOnDesktop() {
+    PostsCreatorDesktop postsCreator = new PostsListPage().open().getPostCreatorDesktop();
+
+    postsCreator.clickPostCreator();
+
+    Assertion.assertTrue(postsCreator.isExpanded());
+    Assertion.assertFalse(postsCreator.isPostButtonActive());
+  }
+
+  @Test(groups = "discussions-loggedInUsersDesktopPosting")
+  @Execute(asUser = User.USER)
+  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+  public void userOnDesktopCannotSaveEmptyPost() {
+    PostsCreatorDesktop postsCreator = new PostsListPage().open().getPostCreatorDesktop();
+
+    postsCreator
+            .clickPostCreator()
+            .closeGuidelinesMessage();
+
+    Assertion.assertFalse(postsCreator.isPostButtonActive());
+  }
+
+  @Test(groups = "discussions-loggedInUsersDesktopPosting")
+  @Execute(asUser = User.USER)
+  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+  public void userOnDesktopCanAddPostWithoutTitle() {
+    PostsListPage postList = new PostsListPage().open();
+    PostsCreatorDesktop postsCreator = postList.getPostCreatorDesktop();
+
+    postsCreator
+            .clickPostCreator()
+            .closeGuidelinesMessage()
+            .fillPostContent("dupakupa123")
+            .selectCategory(0);
+
+    Assertion.assertTrue(postsCreator.isPostButtonActive());
+
+    postsCreator.clickSubmitButton();
+
+    Post post = postList.getPost();
+    post.getNewestPost()
```

Comment:
If new post is not present, test will pass cause of Optional. Please don't use any if/else logic in automated scripts.